include/bluetooth/mesh/access.h

```diff
@@ -467,18 +467,22 @@ struct bt_mesh_model_cb {
 				  size_t len_rd, settings_read_cb read_cb,
 				  void *cb_arg);
 
-	/** @brief Callback called when all settings have been loaded.
+	/** @brief Callback called when the mesh is started.
 	 *
-	 *  This handler gets called after the settings have been loaded in
-	 *  full.
+	 *  This handler gets called after the node has been provisioned, or
+	 *  after all mesh data has been loaded from persistent storage.
 	 *
-	 *  @sa settings_handler::h_commit
+	 *  When this callback fires, the mesh model may start its behavior,
+	 *  and all Access APIs are ready for use.
 	 *
-	 *  @param model Model this callback belongs to.
+	 *  @param model      Model this callback belongs to.
+	 *  @param first_time Flag indicating whether this is the first time the
+	 *                    node is started. If false, provisioning data was
+	 *                    loaded from persistent storage.
```

Comment:
What kind of use cases do you have in mind for this parameter? I.e. what kind of things could the callback decide (or not decide) to do with it?

Reply:
To be honest, I don't have a strong use case for this at the moment. It makes it possible to retain any previous logic in the settings_commit callback, like reconstruction of state based on loaded values (as opposed to using default values the first time around), but I'm not sure if this is problem anyone would be facing. 

I added it because it's potentially interesting information that's otherwise lost after this callback. If you think it just complicates the API or if it's inaccurate somehow, I'm not going to fight for it.